matchbook-sdk-common/pom.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>matchbook-sdk</artifactId>
+        <groupId>com.matchbook</groupId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>matchbook-sdk-common</artifactId>
```

Comment:
I don't like _common_ very much. What about _api_? Or maybe _core_ and the current _core_ renamed to _rest_?

Reply:
yeah, you right. 
I would recommend you then to take a look on that split as on a logical structure, not a the names